README.md

```diff
@@ -1 +1,9 @@
-# samples-java-micronaut
+# Micronaut OAuth Sample Applications for Okta
```

Comment:
If there's only one sample, I'm not sure we should say "applications".

Reply:
planning to add more sub-modules/applications/flows, therefore left it as "applications".